_posts/2024-12-19-cosmicqc.md

```diff
@@ -0,0 +1,285 @@
+
+Since I couldn't find any other paper that focuses on quality control at the single-cell level, I completely agree with their statement. 
+To summarize the paper, they present a cell-level quality control workflow which uses machine learning to detect artifacts within images.
+They utilized CellProfiler to segment single cells and extract features from both the cells and whole images.
+They wanted to group images based on their phenotype, so they utilized CellProfiler features (like `PowerLogLogSlope`, `FocusScore`, `MeanIntensity`, `Cell Count`, etc.) to detect images that contains artifacts versus no artifacts.
+Once images were grouped together, they performed a method using KDE to sample the images into their respective phenotypes.
+They then trained an SVM (Support Vector Machine) per phenotype group, to classify where there are artifacts and where there are cells.
+I believe I barely scratch the surface of the complexities behind this method, so feel free to read the paper if you want a more in-depth look.
+
+What I really liked about this paper is that it is attempting to preserve images that include artifact, as these images still contain other cells.
+It is different than the current process where whole images are removed that fail QC metrics like blur and over-saturation (traditional done in CellProfiler).
+But, I also have my suspicions about using cells that come from an image with corruption, such as an artifact.
+I wonder how much things like an artifact or smudge could be directly impacting the biology the cells.
+In my opinion, I would rather default to losing those cells in an image than risk any issues in downstream interpretation of the results do to any impacts the artifacts in an image had.
+
+I have two main concerns from this paper, specifically for it's generalizability and usability in my own pipelines:
+
+1. It was not clear to me how this method is filtering out artifacts at the single-cell level. From what I understood from the figures, the SVM are able to detect where there is artifacts and where there are cells, but I didn't see where in their pipeline to they remove segmentations from the CellProfiler output. This is a big concern for generalizability as it seems like you would need to train SVMs on your data to be able to apply this method but there isn't a direct link to that output and filtering cells.
+2. The code is available via FigShare, but it was a bit hard to find and parse. Since it is not on GitHub, that also limits the usability since there isn't any environment or dependencies I could find and there isn't a way to create issues and contact the maintainers for assistance. As well, I found that the code I could find was not easy to understand and lacked documentation. I want to make clear, I appreciate that they made the code available and open source which is more than some other papers. My main point is that I don't think this method is feasible to include in my standard workflow.
```

Comment:
```suggestion
2. The code is available via FigShare, but it was a bit hard to parse. Since it is not on GitHub, that also limits the usability since there isn't an environment or dependencies, and there isn't a way to create issues and contact the maintainers for assistance. As well, I found that the code was not easy to understand and lacked documentation. I want to make clear, I appreciate that they made the code available and open source which is more than some other papers! My main point is that I don't think this method is feasible to include in my standard workflow.
```